proj/src/main/webapp/script.js

```diff
@@ -144,24 +144,64 @@ $('#collapse-icon').addClass('fa-angle-double-left');
 
 // Collapse on click
 $('[data-toggle = sidebar-colapse]').click(function() {
-    SidebarCollapse();
+    sidebarCollapse();
 });
 
-// Currently hides the sidebar on smaller and medium screens (TODO: adjust screen for different screen sizes)
-function SidebarCollapse () {
+function sidebarCollapse () {
+    // remove locational reset 
+    $('#bottom').removeClass('topbtn'); 
+
+    // if the device is small, this will hide the chat when they open the side bar
+    if (screen.width < 750) {
+        $('.sidebar + .p-4').toggleClass('d-none'); 
+    }
+
+    // collapse sidebar as normal 
     $('.menu-collapsed').toggleClass('d-none');
     $('.sidebar-submenu').toggleClass('d-none');
     $('.submenu-icon').toggleClass('d-none');
     $('#sidebar-container').toggleClass('sidebar-expanded sidebar-collapsed');
-    
+
     // Treating d-flex/d-none on separators with title
     var SeparatorTitle = $('.sidebar-separator-title');
     if ( SeparatorTitle.hasClass('d-flex') ) {
         SeparatorTitle.removeClass('d-flex');
     } else {
         SeparatorTitle.addClass('d-flex');
     }
-    
-    // Collapse/Expand icon
-    $('#collapse-icon').toggleClass('fa-angle-double-left fa-angle-double-right');
-}
\ No newline at end of file
+
+    // move the buttom on mobile view
+    if ((screen.width<500) && ($( "#sidebar-container" ).hasClass( "sidebar-collapsed" ))) {
+        $('#bottom').addClass('topbtn'); 
+    }
+
+     // Collapse/Expand icon
+     $('#collapse-icon').toggleClass('fa-angle-double-left fa-angle-double-right');
+}
+
+/*  This makes sure that the logic is working properly for the classes.
+    This ensures that if the sidebar is opened on a mobile device, the chat is hidden.
+    It also make sure that there is not an d-none tag on the chat if the sidebar
+    is closed. 
+**/
+function checkLoadingDisplays() {
+    if ($( '#sidebar-container' ).hasClass( 'sidebar-collapsed' )) {
+        $('.sidebar + .p-4').addClass('d-block'); 
+    }
+    else {
+        $('.sidebar + .p-4').addClass('d-none'); 
+    }
+}
+
+$( document ).ready(function() {
+    if (screen.width < 750) {
+        checkLoadingDisplays();
+    }
+
+    // adjust message tempate proportions
+    if (screen.width < 800) {
```

Comment:
These 2 numbers as well

Reply:
800 is for a general tablet view while 750 for tablets smaller than an iPad